content/en/academy/localstack-developement/lesson-3/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "LocalStack Integrations - Infrastructure-as-Code and CI tools "
+linkTitle: "LocalStack Integrations - Infrastructure-as-Code and CI tools "
+weight: 3
+description: >
+  In the last video where we manually deployed some AWS resources, we realized that there were a lot of resources that needed to be deployed and doing it manually can be really hard and prone to errors. We mostly interacted with LocalStack through the CLI. However, large systems are hardly built this way. To the rescue comes LocalStack’s integrations. LocalStack supports a wide range of integrations from the cloud development ecosystem. One such integration is terraform, which is an Infrastructure as Code ( IaC ) tool, which we are going to use to setup the whole AWS infrastructure, that we discussed in the previous video. 
```

Comment:
`terraform` must be properly capitalised (here and in other lessons). Also remove spacing after `(` to avoid the weird line break
![image](https://github.com/localstack/docs/assets/5170829/f6c06acb-adf0-4d20-be95-4036d650c945)